presidio-analyzer/pyproject.toml

```diff
@@ -36,6 +36,7 @@ azure-ai-textanalytics = { version = "*", optional = true }
 azure-core = { version = "*", optional = true }
 transformers = { version = "*", optional = true }
 huggingface_hub = { version = "*", optional = true }
+spacy = "!=3.7.0"  # Exclude spaCy 3.7.0 -> https://github.com/microsoft/presidio/issues/1487
```

Comment:
spacy is already on line 26. Could you please add the 3.7.0 constraint to this line? e.g. `>=3.4.4, <4.0.0, !=3.7.0`